testing/e2tools/APKBUILD

```diff
@@ -2,9 +2,9 @@
 # Maintainer: Daniel Santana <daniel@santana.tech>
 pkgname=e2tools
 pkgver=0.0.16
-pkgrel=0
+pkgrel=1
 pkgdesc="Utilities for manipulating files in an ext2/ext3 filesystem"
-url="http://home.earthlink.net/~k_sheff/sw/e2tools/"
+url="https://manpages.ubuntu.com/manpages/bionic/man7/e2tools.7.html" # original hoempage is dead
```

Comment:
does the source link really work then?

Reply:
is it the same e2tools as this? 

http://e2tools.github.io/
https://github.com/e2tools/e2tools/releases

Reply:
Yes, the source link works but the homepage no

Reply:
OK, source link does not work for me, I get 404